index.html

```diff
@@ -7,160 +7,192 @@
     <title>Telemetry portal</title>
 
     <!-- Bootstrap and supporting libraries -->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
-    <link rel="stylesheet" href="new-pipeline/style/dashboards.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
```

Comment:
Q: I'm not familiar with jsdelivr. What's to recommend it over the "official" bootstrapcdn?

Reply:
this is the bootstrap recommendation https://getbootstrap.com/docs/5.2/getting-started/download/#cdn-via-jsdelivr